### ISSUE TITLE
custom traning dataset?

### ISSUE THREAD

**Issue by ryumansang** [AlexeyAB/darknet#1574]
Nice to meet you.

I am making some custom datasets now.
But the question is, is it possible to learn by classifying the same object into two classes?

Here's an example.

1. Label all car(class 0)
2. Label all lamborghini among car(class0 + class 1)

In this case, can all car be selected and lamborghini can be individually selected?

**Comment by PeterQuinn925** [comment]
What I would do is train with the most specific class and then write a wrapper that applies the parent class. Eg. the wrapper has a list of what child classes (lambo) make up the parent class (car) and and draw the bounding box for the parent class every time darknet detects a child class.

**Comment by AlexeyAB** [comment]
@ryumansang 
Yes, you can. There are two ways, if you use Yolo v3 or Yolo v3 tiny - it depends on what you want to get in detection result:

* mark lamborghini as class_id=1, and other cars as class_id=0

* mark lamborghini with two the same bounded boxes one class_id=0 and second class_id=1 (this is known as multilabeling Training that is supported by Yolo v3), and other cars as class_id=0
